Modules/FlexForm.general.js

```diff
@@ -35,19 +35,24 @@ function showMessage (msg, type, where = false, stick = false) {
 	}
 }
 
+function ffFindFormElementValueByName( form, name ) {
+	if ( name.length > 1 ) {
+		return btoa( $(form).find('select[name="' + atob( name ) + '[]"]').val() );
```

Comment:
If `name` decodes to something like `name" out`,  then `out` will not be part of the `name`-property query. But this does not seem very dangerous.